src/main/java/org/jabref/model/entry/Month.java

```diff
@@ -93,9 +97,28 @@
             int number = Integer.parseInt(value);
             return Month.getMonthByNumber(number);
         } catch (NumberFormatException e) {
-            return Optional.empty();
+            return parseGermanShortMonth(testString);
         }
     }
+    
+    /**
+	 * Parses a month having the string in German standard form such as
+	 * "Oktober" or in German short form such as "Okt"
+	 * 
+	 * @param value,
+	 *            a String that represents a month in German form
+	 * @return the corresponding month instance, empty if input is not in German
+	 *         form
+	 * @author Johannes Pre�mar
+	 */
+	private static Optional<Month> parseGermanShortMonth(String value) {
+		try {
+		YearMonth yearMonth = YearMonth.parse("1969-" + value, DateTimeFormatter.ofPattern("yyyy-MMM", Locale.GERMAN));
```

Comment:
The indent seems to be wrong here. Use auto-format of your IDE. Ctrl+Alt+L in IntelliJ.

Reply:
<s>The idea of https://github.com/JabRef/jabref/pull/3536 was to parse all available languages. Also Spanish, Greek, etc. Therefore, all available locales were fetched at https://github.com/JabRef/jabref/pull/3536/files#diff-3c5fdac594bc2e649536f5ecdcb481ccR122 and iterated on. Is this possible here, too - or does it clutter the code?</s>

After more thinking, I would say, we do not iterate on all locales and keep German only, because only the literature list by http://www2.informatik.uni-stuttgart.de/zdi/buecherei/NCSTRL_listings/FAK/index.html causes these issues and I am not aware of any other systems.

Reply:
I use Eclipse, hopefully Ctrl + Shift + F will have the same result.